src/Bridge/Symfony/Routing/ApiLoader.php

```diff
@@ -48,17 +48,19 @@
     private $operationPathResolver;
     private $container;
     private $formats;
+    private $enableGraphql;
     private $resourceClassDirectories;
     private $subresourceOperationFactory;
 
-    public function __construct(KernelInterface $kernel, ResourceNameCollectionFactoryInterface $resourceNameCollectionFactory, ResourceMetadataFactoryInterface $resourceMetadataFactory, OperationPathResolverInterface $operationPathResolver, ContainerInterface $container, array $formats, array $resourceClassDirectories = [], SubresourceOperationFactoryInterface $subresourceOperationFactory = null)
+    public function __construct(KernelInterface $kernel, ResourceNameCollectionFactoryInterface $resourceNameCollectionFactory, ResourceMetadataFactoryInterface $resourceMetadataFactory, OperationPathResolverInterface $operationPathResolver, ContainerInterface $container, array $formats, bool $enableGraphql, array $resourceClassDirectories = [], SubresourceOperationFactoryInterface $subresourceOperationFactory = null)
```

Comment:
`bool $enableGraphql` must be the last parameter and have a default value or it's a BC break.

Reply:
Could you explain why? It seems to me this is an internal service. I don't see what it could break.

Reply:
not marked as `@internal`

Reply:
Some people (including me :D) use API Platform as a reusable library in other contexts (Silex, Laravel...). This change will break this class of uses.

Reply:
OK thanks for the explanation.